app/dao/notifications_dao.py

```diff
@@ -698,12 +699,29 @@ def dao_old_letters_with_created_status():
     last_processing_deadline = yesterday_bst.replace(hour=17, minute=30, second=0, microsecond=0)
 
     notifications = Notification.query.filter(
-        Notification.updated_at < convert_bst_to_utc(last_processing_deadline),
+        Notification.created_at < convert_bst_to_utc(last_processing_deadline),
         Notification.notification_type == LETTER_TYPE,
         Notification.status == NOTIFICATION_CREATED
     ).order_by(
-        Notification.updated_at
+        Notification.created_at
+    ).all()
+    return notifications
+
+
+def letters_missing_from_sending_bucket(seconds_to_subtract):
+    older_than_date = datetime.utcnow() - timedelta(seconds=seconds_to_subtract)
+
+    notifications = Notification.query.filter(
+        Notification.updated_at == None,  # noqa
```

Comment:
Optional: As your commit message says, it is letters that have not been updated and have no billable units that are missing. Therefore, just to make this very clear in the code it would be nice to have a small code comment saying this. You could also order the query arguments so those two arguments are next to each other and stand out (.e.g above mentioned comment or a line break)